pymongo/monitoring.py

```diff
@@ -16,6 +16,10 @@
 
 .. versionadded:: 3.1
 
+.. attention:: Starting in PyMongo 3.11, the monitoring classes outlined below
```

Comment:
Did you try rendering the docs locally and check that this reference actually works as expected?

Reply:
I did, it shows up as expected.